src/test/resources/org/powermock/extensions/configuration.properties

```diff
@@ -0,0 +1 @@
+powermock.global-ignore=com.sun.org.apache.xerces.*,javax.xml.parsers.*,org.w3c.dom.*,org.xml.sax.*
```

Comment:
Is this like a configuration file for powermock?
If so can we place it under `config` directory and not in source.

Reply:
As I understood it, the config file should be embedded as a resource. It must be in the source tree. Anyway, the Powermock vendors are going to fix it:
https://github.com/powermock/powermock/issues/864#issuecomment-343654812
As soon as the Powermock issue is fixed, this commit should be reverted.

Reply:
Actually, this file can be moved to the folder `resources-noncompilable`.

Reply:
No. The folder `resourrces-noncompilable` won't fit. In this case the file will not be included in the checkstyle-test.jar. This may lead to a problem.

Reply:
They specifically use the classloader, so it must be on the classpath.
See https://github.com/powermock/powermock/blob/4f34a0899446e0a7f5fa5f6f336aa4ab388461ea/powermock-core/src/main/java/org/powermock/configuration/support/PropertiesLoader.java#L36